src/System.Management/tests/System/Management/ManagementClassTestsMofRequired.cs

```diff
@@ -0,0 +1,92 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using Xunit;
+
+namespace System.Management.Tests
+{
+    [SkipOnTargetFramework(TargetFrameworkMonikers.Uap, "WMI not supported via UAP")]
```

Comment:
why not rename IsElevatedAndNotNanoServer to IsElevatedAndSupportsWmi, then you can roll a check for UAP into that, and abstract out what actually supports Wmi.

Reply:
I don't think this `SkipOnTargetFramework(...Uap)` is needed the test project has only netcoreapp and netfx configurations, the tests will not run in Uap.

Reply:
You can actually remove all of the ones disabling for Uap, present in the test classes :)

Reply:
Good idea! I will follow it.

Reply:
I still suggest renaming as I reocmmend above, since then you can adjust what Windows SKUs etc in one place. Eg., if we add IoT.